### PR TITLE
fix: update component status bar to always reflect the active component

### PR DESCRIPTION
**Pull request by macarva**
When the page was being refreshed, the initial state of the component was being set disconsidering the current location, which would reset the state as if it was a fresh navigation.

# Description

## Motivation & context
https://github.com/Microsoft/fast-dna/issues/1303

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by chrisdholt** [comment]
@macarva Updated your PR title to align with our `type : description` pattern

File: packages/fast-development-site-react/src/components/site/index.tsx [hunk 1]
```diff
@@ -371,20 +371,23 @@ class Site extends React.Component<
         super(props);
 
         this.initialPath = this.getInitialPath();
-
+        const path: string =
+            this.initialPath === this.getCurrentPath()
+                ? this.initialPath
```
Comment (nicholasrice) [inline packages/fast-development-site-react/src/components/site/index.tsx]: I think you can just assign this to `this.getCurrentPath()`

Comment (macarva) [reply packages/fast-development-site-react/src/components/site/index.tsx]: Fixed it.

**Comment by macarva** [comment]
Ok I think I fixed what ever happened

**Pull request merged**